costs/tags/tags_keys.go

```diff
@@ -51,12 +52,12 @@ func getTagsKeysWithParsedParams(ctx context.Context, params tagsKeysQueryParams
 		if returnCode == http.StatusOK {
 			return returnCode, response
 		}
-		return returnCode, errors.New("Internal server error")
+		return returnCode, err
```

Comment:
Why didn't you use GetErrorMessage here?

Reply:
`GetErrorMessage`is already called in `makeElasticSearchRequestForTagsKeys`